tile_engine/ops/gemm/gemm_instance_builder.py

```diff
@@ -302,10 +325,13 @@ def _generate_trait_combinations(self):
                 )
         return combinations
 
-    def _generate_kernel_instance(
-        self, tile_config, trait_combo, k_block_per_cu, is_header=True
-    ):
+    def _generate_kernel_instance(self, tile_config, trait_combo):
         """Generate a single kernel instance"""
+
+        k_block_per_cu = self.config.get("k_block_per_cu")
+        if k_block_per_cu is None:
```

Comment:
```suggestion
        k_block_per_cu = self.config.get("k_block_per_cu", 1)
```
dict.get() accepts a second parameter as the default value to return if the key doesn't exist

Reply:
Done